tests/test_tocdirective/test_toc_urllink.html

```diff
@@ -1,9 +1,41 @@
-<div class="tableofcontents-wrapper compound">
-<ul>
-<li class="toctree-l1"><a class="reference internal" href="content1.html">Content 1</a></li>
-<li class="toctree-l1"><a class="reference internal" href="content2.html">Content2</a></li>
-<li class="toctree-l1"><a class="reference internal" href="content3.html">Content3</a></li>
-<li class="toctree-l1"><a class="reference external" href="https://jupyterbook.org/start/build.html">Build your book</a></li>
-<li class="toctree-l1"><a class="reference internal" href="subfolder/index.html">Subfolder</a></li>
-</ul>
-</div>
\ No newline at end of file
+<div class="toctree-wrapper compound">
+ <ul>
+  <li class="toctree-l1">
+   <a class="reference internal" href="content1.html">
+    Content 1
+   </a>
+  </li>
+  <li class="toctree-l1">
+   <a class="reference internal" href="content2.html">
+    Content2
+   </a>
+  </li>
+  <li class="toctree-l1">
+   <a class="reference internal" href="content3.html">
+    Content3
+   </a>
+  </li>
+  <li class="toctree-l1">
+   <a class="reference external" href="https://jupyterbook.org/start/build.html">
+    Build your book
+   </a>
+  </li>
+  <li class="toctree-l1">
+   <a class="reference internal" href="subfolder/index.html">
+    Subfolder
+   </a>
+   <ul>
+    <li class="toctree-l2">
```

Comment:
Note, the `tableofcontents` directive now outputs also sub-lists of the toctree. This is expected, unless the `maxdepth: 1` option is used

Reply:
cc: @AakashGfude we may need to review this in `jupyterbook-latex` support for `tableofcontents` parsing.